playbooks/roles/common-ses/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+radosgw_keystone: False
+
+# Openstack-helm services expects a specific application name which is
+# different from what's created by ses openstack.integrate script.
+# So as interim solution, creating pools with different names to allow
+# exected application names to be enabled with those pools.
+
+socok8s_openstack_glance_pool_name: socok8s_images
+socok8s_openstack_glance_pool_user: glance
+socok8s_openstack_cinder_pool_name: socok8s_volumes
+socok8s_openstack_cinder_pool_user: cinder
+socok8s_openstack_nova_pool_name: socok8s_nova
+socok8s_openstack_libvirt_pool_name: socok8s_vms
+socok8s_openstack_libvirt_pool_user: cinder
+socok8s_openstack_cinder_backup_pool_name: socok8s_cinder_backup
+socok8s_openstack_cinder_backup_pool_user: cinder-backup
+socok8s_default_rbd_pool_name: socok8s_rbd
```

Comment:
I still prefer the os integration salt runner's approach to allow a customizable prefix to make it unique among deployments. In the QE environment at least, SES are shared. It could be true in customer's environment too.  We can make the prefix default to the socok8s workspace prefix, and the user can override it if they choose so.